dev-lang/retro/retro-2021.4.recipe

```diff
@@ -0,0 +1,70 @@
+SUMMARY="Clean, elegant, and pragmatic dialect of Forth"
+DESCRIPTION="Retro is a clean, elegant, and pragmatic dialect of Forth. \
+It provides a simple alternative for those willing to make a break from legacy \
+systems.
+
+The language draws influences from many sources including traditional \
+Forth systems, cmForth, colorForth, Factor, and Parable. It was \
+designed to be easy to grasp and adapt to specific uses.
+
+The basic language is very portable. It runs on a tiny virtual \
+machine (Nga), which is written in C. There are multiple interface \
+options, the main one (rre) is buildable with just the standard C \
+compiler and libraries on most systems (tested at various points \
+on Linux, NetBSD, macOS, and Windows, on x86, x86-64, PPC [emulated], \
+and various ARM processors)."
+HOMEPAGE="http://forthworks.com/retro/"
+COPYRIGHT="2008-2021, Charles Childers"
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="http://forthworks.com/retro/r/RETRO12-$portVersion.tar.gz"
+CHECKSUM_SHA256="9d43aae72527e7b18fdbd3b4237d425b79ad95bb3d0dee6655264970478524b0"
+SOURCE_DIR="RETRO12-$portVersion"
+
+ARCHITECTURES="x86_64 ?x86 ?x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	retro$secondaryArchSuffix = $portVersion
+	cmd:retro$secondaryArchSuffix = $portVersion
+	cmd:retro_describe$secondaryArchSuffix = $portVersion
+	cmd:retro_document$secondaryArchSuffix = $portVersion
+	cmd:retro_embedimage$secondaryArchSuffix = $portVersion
+	cmd:retro_extend$secondaryArchSuffix = $portVersion
+	cmd:retro_muri$secondaryArchSuffix = $portVersion
+	cmd:retro_unu$secondaryArchSuffix = $portVersion
+	cmd:retro_tags$secondaryArchSuffix = $portVersion
+	cmd:retro_locate$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	if [ $targetArchitecture = x86_64 ]
+	then
+		make $jobArgs OPTIONS=-DBIT64 LDFLAGS=-lnetwork
```

Comment:
Please define a variable with "OPTIONS=-DBIT64" as a content for x86_64, then use it in a single make line.

Reply:
Please, forgive my ignorance, but I am not sure I understand what would you like me to do. Could you provide any example in other recipes or otherwise guidance that I can follow?

Reply:
Something like https://github.com/haikuports/haikuports/blob/14c2cab5428145b93232cb69683a67bbe68a9f06/sys-devel/gcc/gcc-2.95.3_2017_07_20.recipe#L82 

local optionArg
if [ $targetArchitecture = x86_64 ]; then
	optionArg="OPTIONS=-DBIT64"
fi
make $jobArgs $optionArg LDFLAGS=-lnetwork

Reply:
fixed